src/AppBundle/Controller/Pa/TeamController.php

```diff
@@ -62,6 +63,10 @@ public function editAction(Request $request, $id)
     {
         $user = $this->getRestClient()->get('team/member/'.$id, 'User');
 
+        if (!$this->getUser()->canEditUser($user)) {
```

Comment:
this check should be also done in the API, that throws exception. So, this could be done by the API throwing access denied exception, caught by the client and re-diplayed, and the logic basically moved from client into the API